### PR TITLE
delete index on delete

### PR DESCRIPTION
**Pull request by MichaelButkovic**
@mparent61 @benghaziboy

### CODE REVIEW

File: djes/signals.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+
+from .models import Indexable
+
+
+@receiver(post_delete)
+def delete_es_index_on_delete(sender, instance, **kwargs):
+    if isinstance(instance, Indexable):
```
Comment (benghaziboy) [inline djes/signals.py]: @mparent61 
just to explain what is going on here,

you can't explicitly state a sender for abstract base classes, so instead we listen to all delete calls and interpret from there. Could be pretty wasteful, but most of our classes are Indexable, so it shouldn't be an issue.

Comment (mparent61) [inline djes/signals.py]: Makes sense to me. 

As for the larger signal-vs-method debate, I agree that our convention should be to minimize signal usage, maybe we just restrict to simple operations on delete (and maybe save() or similar).

**Comment by mparent61** [comment]
LGTM

**Comment by benghaziboy** [comment]
LGTM, cam

**Pull request merged**